crates/vm2/src/single_instruction_test/into_zk_evm.rs

```diff
@@ -333,7 +333,7 @@ impl VmWitnessTracer<8, EncodingModeProduction> for NoOracle {
         &mut self,
         _: u32,
         _: zk_evm::aux_structures::LogQuery,
-        _: zk_evm::aux_structures::PubdataCost,
```

Comment:
I've initially enabled [`unused_qualifications` lint](https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#unused-qualifications), but it seems to be broken; it reports too many false positives to be usable.